app/assets/stylesheets/components.scss

```diff
@@ -4,6 +4,8 @@
   border-radius: 4px;
   color: $white;
   margin: 5px;
+  margin-left: 0;
+  margin-right: 3px;
```

Comment:
Don't modify this, as primary-button used all the places in website.

Reply:
should i use ''display:flex'' (in home-button-container)property for a media break-point(991px) because i cant change the width of the home-main-content
also the result is like this 
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/e28e4f9c-564c-4b2d-96ce-5dd653b99210)
other then the width of the buttons exceeds the ''buttons container'' everything is working fine

Reply:
hey @tanmoysrt  i made the necessary change to be made without changing the ``primary button`` which i did before
so pls pls check the recent commit that i made

Reply:
sure ! I will check